simulator/func_sim/alu.h

```diff
@@ -235,6 +235,7 @@ struct ALU
     template<typename I> static void movz( I* instr)  { move( instr); if (instr->v_src[1] != 0) instr->mask = 0; }
 
     // Bit manipulations
+    template<typename I> static void sbinv( I* instr) { instr->v_dst[0] = instr->v_src[0] ^ ( 1LU << shamt_v_src2<typename I::RegisterUInt>( instr)); }
```

Comment:
`1LU` may be 32 bit, so it shift result is unspecified and Windows has a different results. We have to use `lsb_set<typename I::RegisterUInt>` macro instead.